WORKSPACE

```diff
@@ -25,7 +25,7 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 # Keep this version in sync with:
```

Comment:
This comment is a bit ambiguous now that there's two versions listed. Let's maybe structure it like this:

```python
versions.check(
    # Keep this version in sync ...
    minimum_bazel_version = "4.2.2",
    # TODO(<issue>)...
    maximum_bazel_version = "5.4.0",
)
```

Reply:
I totally overlooked the comments. Thanks for catching this!